tests/bench_xtimer_load/main.c

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2019 Kaspar Schleiser <kaspar@schleiser.de>
+ * Copyright (C) 2019 Freie Universität Berlin
+ * Copyright (C) 2017 HAW Hamburg
+ * Copyright (C) 2015 Eistec AB
+ *               2013 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief    bench_xtimer_load test application
+ *
+ * This is based on tests/xtimer_drift. It removes the lengthy printout of the
+ * worker thread and adds some stats keeping.
+ *
+ * @author   Kaspar Schleiser <kaspar@schleiser.de>
+ * @author   Oliver Hahm <oliver.hahm@inria.fr>
+ * @author   Christian Mehlis <mehlis@inf.fu-berlin.de>
+ * @author   Joakim Nohlgård <joakim.nohlgard@eistec.se>
```

Comment:
Is the email still valid? If I recall correctly, Joakim got a new job recently.

Reply:
Just copied it from xtimer_drift, which this test is based on